plugin/kubernetes/kubernetes.go

```diff
@@ -292,7 +298,8 @@ func (k *Kubernetes) InitKubeCache(ctx context.Context) (onStart func() error, o
 // should be watched, and false when endpoints should be watched.
 // If the API supports discovery v1 beta1, and the server versions >= 1.19, endpointslices are watched.
```

Comment:
This comment should most likely be updated to reflect the new behavior of `endpointSliceSupported`.